src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -558,6 +558,804 @@ def __init__(self, launch_string):
         super().__init__(details)
 
 
+class StandaloneLauncher:
+    """Instantiates Fluent session in standalone mode."""
```

Comment:
Each Launcher class is independent so that we can move them to separate .py files.